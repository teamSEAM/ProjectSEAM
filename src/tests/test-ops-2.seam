#TEST ORDER OF OPS#
entity World:
	func start():
		if((1+2*6)==13):
			screen.out("MULT>ADD PASS")
		else:
			screen.out("MULT>ADD FAIL")

		if((10-6/2)==7):
			screen.out("DIV>SUB PASS")
		else:
			screen.out("DIV>SUB FAIL")

		if(10*24/5)==120):
			screen.out("DIV MULT LR PASS")
		else:
			screen.out("DIV MULT LR FAIL")

		if(((5-3)*5)==10):
			screen.out("COMPLEX PASS")
		else:
			screen.out("COMPLEX FAIL")
