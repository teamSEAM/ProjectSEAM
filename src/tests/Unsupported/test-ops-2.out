MULT>ADD PASS
DIV>SUB PASS
DIV MULT LR PASS
COMPLEX PASS
